content/blog/esc-table-editor-provider-config-support/index.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Introducing the Key-Value Table Editor for Pulumi ESC"
```

Comment:
I would recommend a unique title, instead of the same as: https://www.pulumi.com/blog/esc-key-value-table-editor-launch/. Can we include in the title that provider config is the new part?

Reply:
Ah I forgot to edit that when I copied over the previous blog :)